charts/victoria-metrics-agent/values.yaml

```diff
@@ -76,6 +75,11 @@ affinity: {}
 
 # vmagent scraping configuration:
 # https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/vmagent.md#how-to-collect-metrics-in-prometheus-format
+
+# use existing configmap if specified
+# otherwise .config values will be used
+configMap: ""
```

Comment:
please create a helper which can define what name should be used

Reply:
nice trick, thanks for pointing it